src/components/BSelect/BSelect.vue

```diff
@@ -0,0 +1,284 @@
+<template>
+  <div>
+    <div>
+      <label>{{ selectLabel }} <span v-if=isRequired aria-label='Required'>*</span></label>
+    </div>
+    <div 
+      :aria-expanded="opened"
+      :aria-label="selectLabel"
+      :aria-owns="id"
+      :aria-required="isRequired"
+      :id="'dropdown' + id"
+      @keyup.32.prevent.stop="toggleList"
+      role="combobox"
+      tabindex="0"
+      v-model="selectLabel"
+    >
+      <ul
+        :class="ulClass"
+        :id="id"
+        role="listbox"
+      >
+        <li 
+          :class="dropdownClass" 
+          @click="toggleList"
+          >{{ selectedOption ? selectedOption.text : placeholder }}
+          <span class="dropdown-arrow"><i class="fa fa-caret-down" aria-hidden="true"></i></span>
+        </li>
+        <span v-show="selectErrors.has('selectedOption')" class="error" data-vv-as="selectLabel">{{ selectErrors.first('selectedOption') }}</span>
+        <li
+          v-if=opened
+          v-for="option in options"
+          :aria-selected="selectedOption && selectedOption.id === option.id"
```

Comment:
This might be a good argument for keeping attributes in the template rather than in a computed function. Is it possible to reference the `option` from a computed function like it's referenced here?

Reply:
... wat?

Reply:
you could pass it in as an argument to the function @eleanormal but I'd prefer to just use the template attributes. Everything stays in one place instead of needing to go look for what's happening.